### DISCUSSION TITLE
Ενδιάμεση Παρουσίαση Προόδου

### DISCUSSION THREAD

**Discussion by StylianiAdam** [courses-ionio/sw#1441] [Q&A] [answered]
Καλησπέρα,

Ήθελα να ρωτήσω αν θα προστεθούν επιπλέον ώρες για ενδιάμεση παρουσίαση προόδου, αφού στο calendly δεν υπάρχουν άλλες διαθέσιμες και δεν πρόλαβα να κλείσω. 

Ευχαριστώ εκ των προτέρων!

**Comment by epidrome** [comment, marked as answer]
έχουν προστεθεί, αλλά αν δεν βολεύουν υπάρχουν εναλλακτικές #1408
